sympy/series/series.py

```diff
@@ -6,7 +6,56 @@
 def series(expr, x=None, x0=0, n=6, dir="+"):
     """Series expansion of expr around point `x = x0`.
 
-    See the docstring of Expr.series() for complete details of this wrapper.
+       Parameters
+       ==========
+
+       expr : Expression
+              The expression whose series is to be expanded.
+
+       x    : Symbol
+              It is the variable of the expression to be calculated.
+
+       x0   : Number
+              The value around which ``x`` is calculated. Can be any value
+              from ``-oo`` to ``oo``.
+
+       n    : Integer
+              The number of terms upto which the series is to be expanded.
+
+       dir  : String
+              Optional (default: "+")
+              The series-expansion can be bi-directional. If ``dir="+"``,
+              then (x->x0+). If ``dir="-", then (x->x0-). For infinite
+              ``x0`` (``oo`` or ``-oo``), the ``dir`` argument is determined
+              from the direction of the infinity (i.e., ``dir="-"`` for
+              ``oo``).
+       Examples
+       ========
+
+       >>> from sympy import Symbol, series, tan, oo
```

Comment:
If you are calling it as `f.series` there is no need to9 import series. On the other hand, if the documentation entry is for the wrapper it makes sense to call it using `series(f, ...)`.

Reply:
yes,ok.